extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -88,6 +99,8 @@ export class AmpAd extends AMP.BaseElement {
       /*OK*/insertAmpExtensionScript(this.getWin(), extensionTag, true);
     }
     copyAttributes(this.element, newChild);
+    newChild.setAttribute('layout', 'fill');
```

Comment:
`const extensionTag = networkImplementationTag(type);` Don't quite understand this line of code. And I'm not sure if we will also create a `AmpAd3PImpl` element